.circleci/config.yml

```diff
@@ -331,6 +331,31 @@ jobs:
             docker push "tendermint/tendermint"
             docker logout
 
+  reproducible_builds:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Build tendermint
+          no_output_timeout: 20m
```

Comment:
Does this mean circleci could hang for 20 min?

Reply:
It would not hang, it just does not produce output while building. The build may last at most 20m, circleci default value is 10m. I could do some work upstream for this, though it will take some time.

Reply:
No need